crates/bevy_pbr/src/light_probe/environment_map.wgsl

```diff
@@ -153,13 +161,35 @@ fn environment_map_light(
     let Edss = 1.0 - (FssEss + FmsEms);
     let kD = diffuse_color * Edss;
 
-
     if (!found_diffuse_indirect) {
         out.diffuse = (FmsEms + kD) * radiances.irradiance;
     } else {
         out.diffuse = vec3(0.0);
     }
 
     out.specular = FssEss * radiances.radiance;
+
+    // Clearcoat
+
+#ifdef STANDARD_MATERIAL_CLEARCOAT
+
+    // Calculate the Fresnel term `Fc` for the clearcoat layer.
+    // 0.04 is a hardcoded value for F0 from the Filament spec.
+    let clearcoat_F0 = vec3<f32>(0.04);
+    let Fc = F_Schlick_vec(clearcoat_F0, 1.0, clearcoat_NdotV) * clearcoat_strength;
+    let inv_Fc = 1.0 - Fc;
+
+    // We have to copy `clearcoat` here to work around a Naga bug.
```

Comment:
What bug? Please open an issue with wgpu and link to it here.

Reply:
https://github.com/gfx-rs/wgpu/issues/5593, added a link